tools/windows_buildenv.bat

```diff
@@ -1,6 +1,6 @@
 @ECHO OFF
 SETLOCAL ENABLEDELAYEDEXPANSION
-REM this � is just to force some editors to recognize this file as ANSI, not UTF8.
```

Comment:
The encoding of this file is broken - please make a binary diff.

Reply:
Fixed. This is always a pain. Not sure which editor here is broken.

Reply:
GitHub Diff still marks this line as changed

Reply:
Oh sorry. This is fixed now and I have created https://github.com/mixxxdj/mixxx/pull/11513 in order that this will not happen again.